iBudget/fund/views.py

```diff
@@ -30,11 +31,15 @@ def show_fund(request):
             HttpResponse object.
     """
     user = request.user
+    icon_if_none = \
+        'https://s3.amazonaws.com/family-finance-tracker-static/standard_fund/funds.png'
     if user:
         user_funds = []
         for entry in FundCategories.filter_by_user(user):
             if not FinancialGoal.has_goals(fund_id=entry.id):
-                user_funds.append({'id': entry.id, 'name': entry.name})
+                user_funds.append({'id': entry.id, 'name': entry.name,
+                                   'url': AwsService.get_image_url(entry.icon) if entry.icon
```

Comment:
`url` should be built in a local variable. It will increase readability fo this code.

Reply:
will be corrected!